pages/help/_slug.vue

```diff
@@ -4,33 +4,45 @@
             :title="page.title"
             :text="page.summary"
         />
-        <RichText
-            v-if="page.richText"
-            class="content"
-            :rich-text-content="page.richText"
-        />
-        <DividerWayFinder color="help" />
-        <div
+        
+        <section-wrapper>
+            <RichText
+                v-if="page.richText"
+                :rich-text-content="page.richText"
+            />
+        </section-wrapper>
+        
+        <section-wrapper theme="divider">
+            <DividerWayFinder color="help" />
+        </section-wrapper>
+        
+        <section-wrapper
```

Comment:
make the outer section-wrapper a div: we want the section wrappers on the individual help topic blocks and dividers, this unit is just to run the loop on.